auth_saml/views/res_config_settings.xml

```diff
@@ -4,9 +4,9 @@
     <record id="auth_saml_base_settings_form" model="ir.ui.view">
         <field name="name">auth_saml_base_settings_form</field>
         <field name="model">res.config.settings</field>
-        <field name="inherit_id" ref="base.res_config_settings_view_form" />
+	<field name="inherit_id" ref="base.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='module_auth_oauth']" position="after">
+            <xpath expr="//setting[@id='module_auth_oauth']" position="after">
```

Comment:
If this is the reason why the dependency was added, change the element this would be after.

The following XML might be missing the setting element if that is new in Odoo 17.

Reply:
Yup, that was why. Wanted to change the view as little as possible, but now I made the xpath apply to the block name integration instead.